MythicMobs/Mobs/pinkerium

```diff
@@ -31,3 +31,37 @@ pinkubus:
   LevelModifiers:
   Disguise:
   Skills:
+
+syncophant: 
+  Type: zombie
+  Display: '&d Pink Syncophant&r - &k<mob.level>'
+  Health: 300
+  Damage: 4
+  Armor: 3
+  BossBar:
+  Faction: Pinkerium
+  Mount:
+  Options:
+    AlwaysShowName: true
+	Glowing: true
+	KnockbackResistance: 0.7
+	MovementSpeed: 0.2
+  Modules:
+  AIGoalSelectors:
+  AITargetSelectors:
+  Drops:
+  - exp 125-150 1
+  DropsPerLevel:
+  DamageModifiers:
+  - FIRE -1
+  - LAVA +1
+  - FIRE_TICK 0
+  Equipment:
+  - stick:0
```

Comment:
stick:0, what does this field mean, is it okay for stick to be 0?

Reply:
That refers to a specific hand.